ush/gldas_forcing.sh

```diff
@@ -40,8 +40,8 @@ sdat0=`sh $FINDDATE $sdate d-1`
 cd $xpath
 rm -f fort.* grib.*
 
-pathp1=$DCOMIN/us007003/$sdate/wgrbbul/cpc_rcdas
-pathp2=$DCOMIN/$sdate/wgrbbul/cpc_rcdas
+pathp1=$DCOMIN/prod/$sdate/wgrbbul/cpc_rcdas
```

Comment:
I'd suggest to have pathp3 removed.   I'd like stop collecting the CPC data since they are saved in DUMP already